docker-hello-world-manual-task/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: Docker Hello World Manual Task
+description: A simple Go application packaged as a docker image that prints "Hello, World".
+componentType: manual-task
+buildPreset: docker
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /docker-hello-world-manual-task
+thumbnailPath: /docker-hello-world-manual-task/icon.png
+documentationPath: /docker-hello-world-manual-task/readme.md
+tags:
+- Manual Task
```

Comment:
We don't need this

Reply:
Removed in 5a8e35dd659f2fbcf0cca4bb07ab8830e65c137a